services/groupService.js

```diff
@@ -0,0 +1,74 @@
+import Group from '../models/Group';
+import ApiError from '../error/ApiError';
+import { formatForCreate } from '../formatters/groupFormatter';
+
+export const createGroupService = async (groupData) => {
+    try {
+        const formatedUser = formatForCreate(groupData);
+
+        const { id, name, permissions } = await Group.create(formatedUser, { returning: false });
+
+        return { id, name, permissions };
+    } catch (error) {
+        return ApiError.badRequest(error.errors[0].message);

```

Comment:
Common way it's to throw errors not to return

Reply:
fixed

Reply:
fixed